src/main/java/domain/BonusNumber.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import exception.LottoException;
+import utility.StringUtility;
+
+public class BonusNumber {
+
+    private static final String INVALID_BONUS_NUMBER = "유효하지 않은 보너스 번호입니다.";
+    private final LottoNumber lottoNumber;
+
+    public BonusNumber(String bonusNumber) {
+        validateBonusNumber(bonusNumber);
+        bonusNumber = bonusNumber.trim();
+        lottoNumber = new LottoNumber(Integer.parseInt(bonusNumber));
+    }
+
+    private void validateBonusNumber(String bonusNumber) {
+        validateIsEmpty(bonusNumber);
+        validateIsNumber(bonusNumber);
+    }
+
+    private void validateIsEmpty(String bonusNumber) {
+        if(bonusNumber == null){
+            throw new LottoException(INVALID_BONUS_NUMBER);
+        }
+    }
+
+    private void validateIsNumber(String bonusNumber) {
+        if(!StringUtility.isNumber(bonusNumber)){
+            throw new LottoException(INVALID_BONUS_NUMBER);
+        }
+    }
```

Comment:
요런 부분들은 intelliJ의 코드 정렬 단축키를 적절히 활용해도 좋을 것 같아요!

```suggestion
    private void validateIsEmpty(String bonusNumber) {
        if (bonusNumber == null){
            throw new LottoException(INVALID_BONUS_NUMBER);
        }
    }

    private void validateIsNumber(String bonusNumber) {
        if (!StringUtility.isNumber(bonusNumber)){
            throw new LottoException(INVALID_BONUS_NUMBER);
        }
    }
```

Reply:
코드 컨벤션을 잘 숙지하지 못했던것같아요
코드 포맷팅 기능을 잘 사용해보겠습니다!